gebru/roster.json

```diff
@@ -49,6 +49,9 @@
     {
       "name": "Naomi Emojevbe",
       "github": "nay-nae"
+    }, 
+    {"name": "Thayaan Srisathialingam"
+     "github":"THAYANS"
```

Comment:
You are missing a comma between your properties and the indentation could be improved :)

Reply:
@THAYAANS Can you please make the changes?